wearApp/src/main/java/dev/johnoreilly/confetti/wear/di/AppModule.kt

```diff
@@ -20,21 +20,24 @@ import dev.johnoreilly.confetti.wear.settings.SettingsViewModel
 import dev.johnoreilly.confetti.wear.auth.ConfettiGoogleSignOutViewModel
 import dev.johnoreilly.confetti.wear.speakerdetails.SpeakerDetailsViewModel
 import dev.johnoreilly.confetti.work.WorkManagerConferenceRefresh
-import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.androidx.viewmodel.dsl.viewModelOf
+import org.koin.core.module.dsl.bind
+import org.koin.core.module.dsl.singleOf
+import org.koin.core.module.dsl.withOptions
 import org.koin.dsl.module
 
 @OptIn(ExperimentalHorologistAuthDataApi::class, ExperimentalHorologistAuthUiApi::class)
 val appModule = module {
-    viewModel { SessionDetailsViewModel(get(), get(), get()) }
-    viewModel { SpeakerDetailsViewModel(get(), get()) }
-    viewModel { ConferencesViewModel(get(), get(), get()) }
-    viewModel { SessionsViewModel(get(), get()) }
-    viewModel { HomeViewModel(get(), get(), get()) }
-    viewModel { SettingsViewModel(get(), get()) }
-    viewModel { GoogleSignInViewModel(get(), get<GoogleSignInAuthUserRepository>()) }
-    viewModel { SignInPromptViewModel(get()) }
-    viewModel { ConfettiGoogleSignOutViewModel(get()) }
+    viewModelOf(::SessionDetailsViewModel)
+    viewModelOf(::SpeakerDetailsViewModel)
+    viewModelOf(::ConferencesViewModel)
+    viewModelOf(::SessionsViewModel)
+    viewModelOf(::HomeViewModel)
+    viewModelOf(::SettingsViewModel)
+    viewModelOf(::SignInPromptViewModel)
+    viewModelOf(::ConfettiGoogleSignOutViewModel)
+    viewModelOf(::GoogleSignInViewModel)
     single { GoogleSignIn.getClient(get<Context>(), GoogleSignInOptions.DEFAULT_SIGN_IN) }
-    single { GoogleSignInAuthUserRepository(get(), get()) }
-    single<ConferenceRefresh> { WorkManagerConferenceRefresh(get()) }
+    singleOf(::GoogleSignInAuthUserRepository).withOptions { bind<GoogleSignInEventListener>() }
```

Comment:
`bind` includes the specified type as a secondary resolution target. That means: it can be resolved by `get<GoogleSignInAuthUserRepository>()` and `get<GoogleSignInEventListener>()`. That enables us to use the constructor DSL to resolve `GoogleSignInViewModel` correctly.